WalletWasabi.Fluent/ViewModels/CoinJoinProfiles/ManualCoinJoinProfile.cs

```diff
@@ -0,0 +1,31 @@
+using Avalonia.Media;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using WalletWasabi.Fluent.Helpers;
+
+namespace WalletWasabi.Fluent.ViewModels.CoinJoinProfiles;
+
+public class ManualCoinJoinProfile : CoinJoinProfileViewModelBase
+{
+	public ManualCoinJoinProfile(int minAnonScoreTarget, int maxAnonScoreTarget, int feeTargetAverageTimeFrameHours)
+	{
+		MinAnonScoreTarget = minAnonScoreTarget;
+		MaxAnonScoreTarget = maxAnonScoreTarget;
+		FeeTargetAverageTimeFrameHours = feeTargetAverageTimeFrameHours;
+	}
+
+	public override string Title => "Economy";
```

Comment:
Shouldn't this be:

```suggestion
	public override string Title => "Manual";
```

Reply:
The naming and the description are out of the scope of this PR. It has to be something meaningful and well thought out.

Reply:
Yes!